docs/guides/notifications.md

```diff
@@ -103,19 +103,39 @@ When the verbosity is set to `info`, the controller will alert if:
 
 ![info alert](../diagrams/slack-info-alert.png)
 
-## GitHub commit status
+## Git commit status
 
-The GitHub provider is a special kind of notification provider that based on the
-state of a Kustomization resource, will update the
-[commit status](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks) for the currently reconciled commit id.
+The `github` and `gitlab` provider are slightly different to the other chat providers. These providers will
+link an event back to its source by writing a commit status event to the repository. For more information about how a
+commit status works, refer to the [GitHub](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks)
+or [GitLab](https://docs.gitlab.com/ee/api/commits.html) documentation.
 
-The resulting status will contain information from the event in the format `{{ .Kind }}/{{ .Name }} - {{ .Reason }}`.
+The first image is an example of how it may look like in GitHub while the one below is an example for GitLab.
 ![github commit status](../diagrams/github-commit-status.png)
```

Comment:
Need to update the image used.

Reply:
Let's update the image in a followup PR.